src/shared/codelens/pythonCodeLensProvider.ts

```diff
@@ -176,19 +175,27 @@ const makeDebugConfig = ({ debugPort, samProjectCodeRoot }: {
                 remoteRoot: '/var/task'
             }
         ],
+        // Disable redirectOutput to prevent the Python Debugger from automatically writing stdout/stderr text
+        // to the Debug Console. We're taking the child process stdout/stderr and explicitly writing that to
+        // the Debug Console.
+        redirectOutput: false,
     }
 }
 
 export async function initialize({
     configuration,
     outputChannel: toolkitOutputChannel,
     processInvoker = new DefaultSamCliProcessInvoker(),
-    taskInvoker = new DefaultSamCliTaskInvoker(),
     telemetryService: telemetryService,
+    ...initializeParams
```

Comment:
Nit: You're only using `localInvokeCommand` from `initializeParams`, might as well inline it here instead.